scalacheck/test/src/peschke/UnitSpec.scala

```diff
@@ -6,23 +6,16 @@ import org.scalatest.OptionValues
 import org.scalatest.matchers.must.Matchers
 import org.scalatest.prop.TableDrivenPropertyChecks
 import org.scalatest.propspec.AnyPropSpec
-import org.scalatest.wordspec.AnyWordSpec
 import org.scalatestplus.scalacheck.ScalaCheckDrivenPropertyChecks
 
 trait CommonSpec extends Matchers with EitherValues with OptionValues
 
-trait UnitSpec extends AnyWordSpec with CommonSpec
-
 trait PropSpec
     extends AnyPropSpec
-    with CommonSpec
-    with ScalaCheckDrivenPropertyChecks {
-  implicit def noShrink[T]: Shrink[T] = Shrink.shrinkAny
-}
-
-trait TableSpec
-    extends AnyWordSpec
```

Comment:
Any reason you don't want to extend from `UnitSpec` directly?

**`[need-clarification]`**

Reply:
IIRC `AnyWordSpec` and `AnyPropSpec` are mutually exclusive, though I might test that assumption at some point